test/Microsoft.NET.Build.Containers.IntegrationTests/EndToEndTests.cs

```diff
@@ -291,7 +307,7 @@ public async Task EndToEnd_MultiProjectSolution()
         commandResult.Should().HaveStdOutContaining("Pushed image 'consoleapp:latest'");
     }
 
-    [DockerAvailableTheory(Skip = "https://github.com/dotnet/sdk/issues/42850")]
+    [DockerAvailableTheory(Skip = "aiting for the net10.0 container images.")]
```

Comment:
typo

Reply:
Please create an issue describing the problem with the base images targeting .net10 and describing why this test cannot be fixed with the workaround and link it in this skip